### ISSUE TITLE
failed to load javascript due to MIME type of "text/plain"

### ISSUE THREAD

**Issue by bowie7070** [buildbot/buildbot#7998]
Hi, 

I've just installed buildbot 4.0.2 on windows via pip in a virtual environment and I have gotten a basic configuration up and running. But I get a blank page when I view web server. Checking the developer tools for Google Chrome I get the following message:

index-c14161c8.js:1 Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/plain". Strict MIME type checking is enforced for module scripts per HTML spec.

Please note that I get similar behaviour and error messages for Google Chrome, Mozilla Firefox, and Microsoft Edge browsers.

Is there anything I can do resolve this? Or is there any additional information required than I can provide to help fix this?

**Comment by tdesveaux** [comment]
Fairly sure this is not a Buildbot issue, but a Windows one.

I had this with different framework when working on Windows. It uses Registry entries for filetypes to define the mime type, but the Registry is broken by ... something.

See a similar issue with flask https://github.com/pallets/flask/issues/1045

**Comment by bowie7070** [comment]
I think you're right that it's not buildbot that is at fault here.

I tried installing buildbot 3.11.7 (using pip and in a new virtual environment) and that version worked. Then I tried the version of buildbot 4.0.2 from conda (using conda and not in a virtual environment) and that worked. So it points to some issue in the way the dependencies/environment is configured.

I'm not sure how to narrow down the cause and if it's not happening widely, it's probably not worth further investigation.
